package.json

```diff
@@ -11,7 +11,7 @@
     "docker:db": "docker-compose exec pg psql -U postgres -d dev-directory",
     "docker:deps": "docker-compose up -d",
     "heroku-postbuild": "npm run build && npx sequelize-cli db:migrate",
-    "jest": "cross-env NODE_ENV=test npx jest --runInBand",
+    "jest": "cross-env NODE_ENV=test npx jest --runInBand --forceExit",
```

Comment:
I could not get this to pass github actions testing without it. It would hang otherwise.